### PR TITLE
Added clarification to Management Requirements

### PR DESCRIPTION
**Pull request by PhillyUrbs**
also made grammar modifications.

### CODE REVIEW

File: windows-365/enterprise/requirements.md [hunk 1]
```diff
@@ -62,12 +62,13 @@ If you choose to provision Hybrid Azure AD joined Cloud PCs, then the following
...
 
-- You must have an Intune license so that you can use Intune to manage the devices.
+- You must have an Intune license to use Intune to manage the devices.
 - Users must have licenses for Windows, Intune, Azure AD, and Windows 365 to use their Cloud PC.
 
 ## Management requirements
 
-You must use [Microsoft Endpoint Manager admin center](https://admin.microsoft.com/) to manage your Cloud PCs.
+- You must use [Microsoft Endpoint Manager admin center](https://admin.microsoft.com/) to manage your Cloud PCs.
+- You mus have a Windows 365 Enterprise license to manage Cloud PC configurations.
```
Comment (JohanFreelancer9) [inline windows-365/enterprise/requirements.md:71]:
```suggestion
- You must have a Windows 365 Enterprise license to manage Cloud PC configurations.
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @PhillyUrbs, kindly commit the suggested change. Many thanks.

**Comment by PhillyUrbs** [comment]
@JohanFreelancer9 Thanks for catching that :)

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by ErikjeMS** [comment]
#sign-off

**Comment by ErikjeMS** [comment]
#sign-off

**Comment by PhillyUrbs** [comment]
@ErikjeMS For some reason this want's another review from you.

**Comment by ErikjeMS** [comment]
Since I signed off, it should be in the queue for merging. As a MS employee, next time you should make your changes in the private branch. Things run smoother there.

**Review by ErikjeMS — APPROVED** [review]

**Comment by ErikjeMS** [comment]
#sign-off

**Pull request merged**